### PR TITLE
Add a method to ProgramService to allow adding a block with questions

### PR DESCRIPTION
**Pull request by carolinedanzi**
### Description
We'll want the ability to add a block with name, description and questions

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java [hunk 1]
```diff
@@ -69,18 +69,34 @@ public ProgramDefinition updateProgramDefinition(long programId, String name, St
...
 
-    BlockDefinition blockDefinition =
+    BlockDefinition.Builder blockDefinitionBuilder =
         BlockDefinition.builder()
             .setId(blockId)
             .setName(blockName)
-            .setDescription(blockDescription)
-            .build();
+            .setDescription(blockDescription);
+    if (!questionDefinitions.isEmpty()) {
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java:91]: nit: does it matter if `questionDefinitions` is empty or not? Couldn't we `setQuestionDefinitions(questionDefinitions)` either way?

**Comment by kendrickt** [comment]
looks good to me!

**Review by kendrickt — APPROVED** [review]
Looks good to me!

File: universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java (hunk 1 above)
Comment (carolinedanzi) [reply universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java]: thank you for calling that out, fixed!

**Pull request merged**